datalad/distribution/create.py

```diff
@@ -264,29 +280,31 @@ def __call__(
             # comes around
             gitattr.write('** annex.largefiles=nothing\n')
 
-        # save everthing
-        tbds.add('.datalad', to_git=True, save=False)
-
-        if save:
-            save_dataset(
-                tbds,
-                paths=['.datalad'],
-                message='[DATALAD] new dataset')
-
-            # the next only makes sense if we saved the created dataset,
-            # otherwise we have no committed state to be registered
-            # in the parent
-            if dataset is not None and dataset.path != tbds.path:
-                # we created a dataset in another dataset
-                # -> make submodule
-                dataset.add(tbds.path, save=save, ds2super=True)
-
-        return tbds
+        # save everything
+        tbds.add('.datalad', to_git=True, save=save,
+                 message='[DATALAD] new dataset')
+
+        # the next only makes sense if we saved the created dataset,
+        # otherwise we have no committed state to be registered
+        # in the parent
+        if save and dataset is not None and dataset.path != tbds.path:
+            # we created a dataset in another dataset
+            # -> make submodule
+            dataset.add(tbds.path, save=save, ds2super=True)
+
+        yield get_status_dict(
+            action='create',
+            ds=tbds,
+            logger=lgr,
+            refds=refds_path,
+            status='ok')
 
     @staticmethod
-    def result_renderer_cmdline(res, args):
+    def custom_result_renderer(res, **kwargs):
         from datalad.ui import ui
-        if res is None:
+        if res.get('action', None) == 'create' and \
+               res.get('status', None) == 'ok' and \
+               res.get('type', None) == 'dataset':
+            ui.message("Created dataset at {}.".format(res['path']))
+        else:
             ui.message("Nothing was created")
```

Comment:
ah... I was running `datalad get --help` as to see options pertaining to that command.  I guess, ideally, we should adjust our `datalad COMMAND --help` to state that there could also be `datalad` options , i.e. have something like
```
Usage: datalad [DATALAD_OPTIONS] clone [--version] [-h] [-l LEVEL] [--pbs-runner {condor}]
                     [-d DATASET] [-D DESCRIPTION] [--reckless]
                     SOURCE [PATH]
....
* DataLad options*
   See `datalad --help` for the full description of the options available to any datalad command [not sure if we might want to at least list them all over here]
```
and may be strip away `--log-level` and `--pbs-runner` to be replicated in `COMMAND --help`.